src/lib/utilities/trim-trailing-slash.ts

```diff
@@ -0,0 +1,3 @@
+export const trimTrailingSlash = (x: string): string => {
+  return x.endsWith('/') ? x.slice(0, -1) : x;
```

Comment:
We could also do something like this to trim all trailing slashes (e.g. for something like `http://cats.meow//`)
```suggestion
  return x.replace(/\/+$/, "")